typescript/web/src/components/website/ukrainian-logo.tsx

```diff
@@ -0,0 +1,76 @@
+import { chakra, HTMLChakraProps, useColorModeValue } from "@chakra-ui/react";
+import * as React from "react";
+
+export const UkrainianLogo = React.forwardRef<
+  SVGSVGElement,
+  HTMLChakraProps<"svg"> & { iconColor?: string }
+>((props: { iconColor?: string }, ref) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { iconColor = "brand", ...rest } = props;
+  return (
+    <chakra.svg
```

Comment:
Except from the viewbox ternary, it looks like this file is now exactly the same than the other Logo. Can we merge both?